### DISCUSSION TITLE
Soc_Score use case

### DISCUSSION THREAD

**Discussion by pimpernoni** [Security-Onion-Solutions/securityonion#10796] [General]
I am reviewing some of the content on various alerts and I note that an "Soc_Score" accompanies many of them, with some metric that varies between alerts. However, I cannot find any description in the forum or documentation that describes this score, how it is calculated, and its intended application.

Can anyone illuminate this?

Thanks

P

**Comment by eb-op** [comment]
Hello pimpernoni! If I am understanding it correctly, the soc_score field in alerts come from Elasticsearch's _score function in queries. This is a float to determine how relevant a document is to a query. I hope this helps!
